### ISSUE TITLE
3.9.0 Migration Guide is missing re-locations for Rest Client Classic

### ISSUE THREAD

**Issue by josephearl** [quarkusio/quarkus#39708]
### Description

The migration guide has the new extension and configuration property names for the Reactive extensions https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.9

In the case of rest client, the reactive version has moved to the coordinates of the classic/blocking version (quarkus-rest-client-reactive -> quarkus-rest-client).

The guide does not say what quarkus-rest-client has become nor it's configuration properties (it looks like it has become quarkus-resteasy-client)

**Comment by gsmet** [comment]
We actually did the change in two steps because of this: we wanted to make sure people would notice the change for the RESTEasy Classic REST Client first because the artifacts would be reused in 3.9.

That's why we did it in 3.7 and the change was documented in the 3.7 migration guide: https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.7#rest-client-gear-white_check_mark .

Note that all of this is handled by the `quarkus update` so I would recommend using it to update your apps.

Let me know if you have any further questions/issues with this update, as it's an important one and we want it to be as smooth as possible.
